dataland-backend/build.gradle.kts

```diff
@@ -69,3 +69,12 @@ jacoco {
     toolVersion = "0.8.7"
     applyTo(tasks.bootRun.get())
 }
+
+tasks.register<Copy>("getTestData") {
+    from("$rootDir/testing/data/companies.json")
```

Comment:
Kleinigkeit, aber wollen wir vllt die Test-Daten auch danach benennen, was sie beinhalten?

D.h. statt ´companies.json´ lieber ´companyInformation.json´

Reply:
Beide Dateinamen geändert in CompanyInformation.json und CompanyAssociatedEuTaxonomyData.json